docs/developer-guide/emt-architecture-overview.md

```diff
@@ -46,16 +46,32 @@ outlines the key differences between those.
 ### Developer Node mutable ISO image
 
 Mutable Developer node in an ISO format allows you to add packages and
-customize the system after deployment. During installation you can select one of four versions:
+customize the system after deployment. During setup you can select one of four versions:
 
 - Standard kernel
-- Standard kernel with Docker and K3S included
+- Standard kernel - Docker and K3S provisioned during installation
 - [Kernel with real-time extensions](#preempt-rt-kernel)
-- [kernel with real-time extensions](#preempt-rt-kernel), with Docker and K3S included
+- [Kernel with real-time extensions](#preempt-rt-kernel) - Docker and K3S provisioned during installation
```

Comment:
Docker not installed in immutable image.
k3s and extensions are installed during provisioning in immutable image.